Realm/RLMRealm.h

```diff
@@ -91,6 +93,26 @@ NS_ASSUME_NONNULL_BEGIN
 + (instancetype)realmWithURL:(NSURL *)fileURL;
 
 /**
+ Open a Realm asynchronously on a background queue, and then invoke a callback
+ on success or failure.
+
+ Opening a Realm asynchronously may be useful if, for example, a time-consuming
```

Comment:
"may be useful if, for example," is needlessly hesitant about what this is for. A rough stab at better wording:

Opening a Realm asynchronously will perform all work needed to get the Realm to a usable state (such as running potentially time-consuming migrations) on a background thread before dispatching to the given queue. In addition, synchronized Realms wait for all remote content available at the time the operation began to be downloaded.